src/Microsoft.AspNet.Mvc.Core/AntiForgery/AntiForgeryWorker.cs

```diff
@@ -107,11 +111,11 @@ public TagBuilder GetFormInputElement([NotNull] HttpContext httpContext)
             SaveCookieTokenAndHeader(httpContext, newCookieToken);
 
             // <input type="hidden" name="__AntiForgeryToken" value="..." />
-            var retVal = new TagBuilder("input");
-            retVal.Attributes["type"] = "hidden";
-            retVal.Attributes["name"] = _config.FormFieldName;
-            retVal.Attributes["value"] = _serializer.Serialize(formToken);
-            return retVal;
+            var inputTag = new TagBuilder("input", _htmlEncoder);
+            inputTag.Attributes["type"] = "hidden";
```

Comment:
@pranavkm not part of this PR and conceptually poor because the `TagBuilder` is not its attributes.  anyhow, if you want pretty, do
``` c#
  var inputTag = new TagBuilder("input", _htmlEncoder)
  {
    Attributes =
    {
      ["type"] = "hidden",
      ...
    },
  };
```

@ajaybhargavb syntax choice here is up to you.  if you choose to use init syntax, also completely up to you whether to use `["type"] = "hidden",` or `{ "type", "hidden" },`.  it's all the same.